packages/@react-spectrum/table/test/TableSizing.test.tsx

```diff
@@ -1258,9 +1258,7 @@ describe('TableViewSizing', function () {
 
         fireEvent.keyDown(document.activeElement, {key: 'Enter'});
         fireEvent.keyUp(document.activeElement, {key: 'Enter'});
-        expect(onResizeEnd).toHaveBeenCalledTimes(1);
-        expect(onResizeEnd).toHaveBeenCalledWith(new Map<string, ColumnSize>([['foo', 600], ['bar', '1fr'], ['baz', '1fr']]));
-
```

Comment:
The changes made in this PR mean that onResizeStart/End aren't triggered unless a resize operation actually happens. Entering and leaving resize mode without changing the size doesn't cause those event to fire. 

Open to opinions on this change. Note that If we want to preserve this behavior then we'll need a new prop like in https://github.com/adobe/react-spectrum/pull/4061/commits/a6912801fc637a70bb96e8d9f08ea8324dd2a48e so we can start resizing when the resizer is focused by some external action